src/cgame/cg_rocket_draw.cpp

```diff
@@ -917,6 +917,46 @@ class SpeedGraphElement : public HudElement
 	Color::Color backColor;
 };
 
+class PositionElement : public HudElement
```

Comment:
Use TextHudElement

Reply:
What is the difference? Should it also be applied to SpeedGraphElement? I guess not, since it can (only in option 3 IIRC) enable a graph, but could you details the why?

Reply:
That would be because it avoids code such as

```C++
currentPositionElement = dynamic_cast< Rml::Core::ElementText* >( AppendChild( Rml::Core::Factory::InstanceElement(
			this,
			"#text",
			"span",
			xml) ) );
```

Reply:
Yes, it avoid the boiler plate of having to instance and append a new child. You can just call SetText() internally.